### PR TITLE
Update Linux Get-Dependency script

### PR DESCRIPTION
**Pull request by davidmoore1**
The linux getDependencies-Linux.sh script is messed up in a way
similar to what was an issue with the windows one.  Files are being copied in from
release and then being overwritten later by copies from the debug directories.

### CODE REVIEW

File: build/getDependencies-Linux.sh [hunk 1]
```diff
@@ -172,13 +172,6 @@ copy_auto http://build.palaso.org/guestAuth/repository/download/bt323/latest.las
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt323/latest.lastSuccessful/ChorusHubApp.exe ../lib/dotnet/ChorusHubApp.exe
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt323/latest.lastSuccessful/ChorusHubApp.exe.mdb ../lib/dotnet/ChorusHubApp.exe.mdb
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt323/latest.lastSuccessful/ChorusMerge.exe ../lib/dotnet/ChorusMerge.exe
-copy_auto http://build.palaso.org/guestAuth/repository/download/bt323/latest.lastSuccessful/ChorusMerge.exe.mdb ../lib/dotnet/ChorusMerge.exe.mdb
```
Comment (StephenMcConnel) [inline build/getDependencies-Linux.sh]: Why did you delete the .mdb file here?  It doesn't seem to be coming from the /debug/ folder.

File: build/getDependencies-Linux.sh [hunk 2]
```diff
@@ -236,8 +222,6 @@ copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.las
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.lastSuccessful/Enchant.Net.dll.config ../lib/dotnet/Enchant.Net.dll.config
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.lastSuccessful/Ionic.Zip.dll ../lib/dotnet/Ionic.Zip.dll
 copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.lastSuccessful/L10NSharp.dll ../lib/dotnet/L10NSharp.dll
-copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.lastSuccessful/NDesk.DBus.dll ../lib/dotnet/NDesk.DBus.dll
-copy_auto http://build.palaso.org/guestAuth/repository/download/bt322/latest.lastSuccessful/NDesk.DBus.dll.config ../lib/dotnet/NDesk.DBus.dll.config
```
Comment (StephenMcConnel) [inline build/getDependencies-Linux.sh]: Why did you delete these NDesk.Dbus files?  They don't seem to be coming from a /debug/ folder.

**Comment by StephenMcConnel** [comment]
There appear to be 3 lines too many deleted from the file.  Until these are restored, or the deletion explained, I can't approve this changeset.  :-1:

**Comment by davidmoore1** [comment]
The chorus mdb file deletion was an error, which has been corrected, which shows why we have the sign off required I guess. Sorry.  The other two lines for NDesk were where it was being copied from bt322.  At line 182, the same two files were already copied from bt323.  It is rebuilt so rarely that it probably doesn't make a difference I suppose, but it didn't look right either.  I'll put them back in because I don't think it really matters, but it does look like some sort of error in the script.

**Comment by StephenMcConnel** [comment]
This has been superseded by https://github.com/BloomBooks/BloomDesktop/pull/390.
